ssl.go

```diff
@@ -105,16 +109,15 @@ func sslClientCertificates(tlsConf *tls.Config, o values) {
 
 // sslCertificateAuthority adds the RootCA specified in the "sslrootcert" setting.
 func sslCertificateAuthority(tlsConf *tls.Config, o values) {
-	if sslrootcert := o.Get("sslrootcert"); sslrootcert != "" {
+	if sslrootcert, ok := o["sslrootcert"]; ok {
```

Comment:
As I understand it, this change breaks compatibility with `libpq` connection strings.

If I'm reading `libpq` source correctly, the CA certificate is only loaded when this setting is not blank:

- [`if (conn->sslrootcert && strlen(conn->sslrootcert) > 0)`](https://github.com/postgres/postgres/blob/REL9_6_2/src/interfaces/libpq/fe-secure-openssl.c#L950-L957)